engine/runtime/src/main/java/org/enso/interpreter/node/callable/IndirectInvokeCallableNode.java

```diff
@@ -13,8 +13,8 @@
 import org.enso.interpreter.runtime.Context;
 import org.enso.interpreter.runtime.callable.UnresolvedSymbol;
 import org.enso.interpreter.runtime.callable.argument.CallArgumentInfo;
-import org.enso.interpreter.runtime.callable.atom.Atom;
-import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+import org.enso.interpreter.runtime.data.struct.Struct;
+import org.enso.interpreter.runtime.data.struct.AtomConstructor;
```

Comment:
More than sixty changed files! I was wondering why the change is so huge. Consider doing the rename of `Atom` to `Struct` as a separate PR.